lib/ansible/module_utils/urls.py

```diff
@@ -481,29 +483,37 @@ def redirect_request(self, req, fp, code, msg, hdrs, newurl):
             elif follow_redirects in ['no', 'none', False]:
                 raise urllib_error.HTTPError(newurl, code, msg, hdrs, fp)
 
+            method = req.get_method()
+
             do_redirect = False
             if follow_redirects in ['all', 'yes', True]:
                 do_redirect = (code >= 300 and code < 400)
 
             elif follow_redirects == 'safe':
-                m = req.get_method()
-                do_redirect = (code >= 300 and code < 400 and m in ('GET', 'HEAD'))
+                do_redirect = (code >= 300 and code < 400 and method in ('GET', 'HEAD'))
 
             if do_redirect:
-                # be conciliant with URIs containing a space
+                # Be conciliant with URIs containing a space
                 newurl = newurl.replace(' ', '%20')
                 newheaders = dict((k, v) for k, v in req.headers.items()
-                                  if k.lower() not in ("content-length", "content-type"))
+                                  if k.lower() not in ("content-length"))
```

Comment:
Why are we modifying this line, and then not using it later?

Reply:
Same question as before.